platform/src/test/java/platformfeatures/PF_Membership.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Validate buttons on member plan page
+
+@MembershipPlan
+Scenario Outline: Validate all the buttons on member plan page
+
+Given Navigate to URL<browser> 
+Then Login To PF Portal 
+Then Click on Membership Status and Click on apply for prime membership banner
```

Comment:
User actions can be prefix with When. Mulitple When statements can be written using And
Eg 
When User clicks on Membership status
And user clicks on apply for membership banner

Reply:
@csonparote Thank you for the suggestions.
I have reframed feature file as you have suggested.

Reply:
Done changes.